tsconfig.json

```diff
@@ -9,24 +9,16 @@
     "declaration": true,
     "noImplicitAny": true,
     "removeComments": false,
-    "noLib": true,
     "preserveConstEnums": true,
     "suppressImplicitAnyIndexErrors": false,
     "outDir": "lib",
     "rootDir": "src"
   },
-  "filesGlob": [
-    "node_modules/typescript/lib/lib.core.es7.d.ts",
-    "custom_typings/*.d.ts",
-    "src/**/*.ts",
-    "typings/index.d.ts"
-  ],
   "files": [
```

Comment:
Please try to remove `files` before submitting. One file is replaced by compilerOptions.lib, the other should go in includes.